### ISSUE TITLE
ComputerCraft is crashing in dev env

### ISSUE THREAD

**Issue by loordgek** [SquidDev-CC/ComputerCraft#4]
when i run cc on its own it is not crashing 

log
https://gist.github.com/loordgek/df6c92729e2344d3445497271e55aafd

build gradle
https://gist.github.com/loordgek/02449ee6d8b4d05c3ae8341692f5b1e9

**Comment by loordgek** [comment]
is the CC on maven obfuscated ??

**Comment by SquidDev** [comment]
Yes, though it shouldn't matter unless you want to read the source - it'll be be deobfuscated at runtime. 

I've no clue if it'll make a difference, but the error doesn't make sense - I'm depending on that build without problem in other mods.

**Comment by loordgek** [comment]
> Can you use compile rather than deobfCompile - CC targets an ancient mappings version, so that might be the cause of the issue.

this not matter then because it will deobfuscated at runtime w/ my mapping version

**Comment by SquidDev** [comment]
If you use `deobfCompile` then it is deobfuscated at dependency resolution time. It shouldn't make a difference, but I can't see a reason fields are magically disappearing either, so it's an least worth a shot.

**Comment by loordgek** [comment]
still the same
https://gist.github.com/loordgek/7245b95a542b1197d0c08ae9ebd1d86e
